### ISSUE TITLE
Filters

### ISSUE THREAD

**Issue by tobbeanderberg** [agical/friendsunhosted#18]
Since it is possible already to tag your updates with #tags (something that we simply have to start doing) a natural feature would be to have filters (better than search) to filter your own stream. Activating a filter would be sort of like an equivalent to a FB-page.

A follow up feature would then be to enable filters to be stored for later reuse. This would be an equivalent to FB-likes.

The final step would be to distribute the filters (giving users the possibility to see what filters their friends have stored). That way you would see your friends likes and interests.

**Comment by jancborchardt** [comment]
I’m actually not sure about tags. As we see from Identi.ca, Twitter and Diaspora, this can lead to #tag spam. Also, tagging is extra work people need to do to make stuff computer-readable. It’s not very pleasant to read »Message text #tag #tag #tag« or »Message about #stuff and #things«.

Instead, a simple stream search would be a good filter (it’s most annoying that Twitter can’t do that).

**Comment by brolund** [comment]
I agree we should be able to filter/search, and I think filter streams would be great. Since you always filter stuff locally anyway, we actually don't need the tags for a central server to understand what to index. Less messy text than e.g. twitter, more powerful searches (in your accessible data).
